src/sentry/runner/__init__.py

```diff
@@ -1,11 +1,16 @@
+import logging
 import os
 import click
 import sys
 import sentry
 import datetime
+import sentry_sdk
 from sentry.utils.imports import import_string
 from sentry.utils.compat import map
 
+logging.basicConfig(format="%(asctime)s [%(levelname)s] %(message)s", level=logging.INFO)
```

Comment:
I've tried to understand how other pieces of code do logging.

I tried using `setLevel(logging.INFO)` on my logger so I could see `logger.info("We have reported the error below to Sentry")` at the end with no avail. Adding `basicConfig()` does the trick but I don't know why it would need that.

I've also tried using logging practices for the project but it's unclear to me.
I've looked at the modules that generate this output:
```
20:11:30 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured.
20:11:32 [INFO] sentry.plugins.github: apps-not-configured
```
and I know they listen to `SENTRY_LOG_LEVEL=INFO` and `SENTRY_LOG_FORMAT='MACHINE'`, however, their code seems typical `import logging` and using a `getLogger()`; what am I missing?